lib/sender.js

```diff
@@ -81,13 +97,24 @@ class Sender {
 
     if (!options.mask) return [target, data];
 
-    randomFillSync(mask, 0, 4);
+    if (randomPool.length - randomPoolIdx < 4) {
+      randomPool = randomBytes(32);
```

Comment:
I'm not convinced this is much better than the status quo. It would only take 8 `WebSocket` each sending a single message to empty the pool and there is a new `Buffer` allocation every time.

Reply:
This is just the fallback path. We shouldn't hit this often.

Reply:
I think you are misunderstanding something here...?